viewing_party/party.py

```diff
@@ -1,23 +1,210 @@
-# ------------- WAVE 1 --------------------
+# # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+    new_movie = create_movie(movie["title"],movie["genre"],movie["rating"])
+    user_data["watched"] = [new_movie]
+
+    return user_data

```

Comment:
You do not need to create a new movie because the movie is being passed in as an argument. In this case we are "pretending" the data already exists. So you would just do 
```python
 # add movie to what user has watched
    user_data["watched"].append(movie)
    return user_data
```